README.md

```diff
@@ -48,19 +48,45 @@ spin build --up
 
 ## Trigger Configuration
 
-The trigger type is `cron` and there are no application-level configuration options. 
+The trigger type is `cron`, and there are no application-level configuration options. 
 
 The following options are available to set in the [[trigger.cron]] section:
 
 | Name                  | Type             | Required? | Description |
 |-----------------------|------------------|-----------|-------------|
 | `component`           | string or table  | required  | The component to run on the schedule given in `cron_expression`. (This is the standard Spin trigger component field.) |
-| `cron_expression`     | string           | required  | The `cron` expresison describing the schedule on which to execute the component. |
+| `cron_expression`     | string           | required  | The `cron` expression describes the schedule for executing the component. |
+
+### Cron Expression Fields
+
+The `cron_expression` format for scheduling is as follows:
+
+```text
+sec   min   hour   day of month   month   day of week   year
```

Comment:
This isn't actually part of the expression is it?

Reply:
Ah, yes. I have updated so that we refer to the `sec`, `min` etc. as fields. I also use a bit of ascii-art inspired text to show each of the fields.